### ISSUE TITLE
[Bug]: Odd/incorrect/broken water textures in Jungle reverse

### ISSUE THREAD

**Issue by k0pp0** [emoose/OutRun2006Tweaks#146]
### Tweaks version

OutRun2006Tweaks-0.6.1

### Describe your issue here (please attach the OutRun2006Tweaks.log file with your report, you can drag+drop it into the text area)

![image](https://github.com/user-attachments/assets/c23402e7-286f-42f3-b624-d126bb6460c1)
![image](https://github.com/user-attachments/assets/afa00bf6-8925-4f71-91e4-97a7c8cf38a3)

comparison shots (thanks Habenero)
![image](https://github.com/user-attachments/assets/b6da5458-b257-4917-aeac-92d4df116e74) <--working
![image](https://github.com/user-attachments/assets/09ee9a77-2447-48f6-8287-d1f31f092f0c) <-not working

**Comment by lance8765** [comment]
Blame the climate change! (jk)

**Comment by Tenome** [comment]
Also worth mentioning that on the Reverse course, the water isn't animated. It just loads what I assume to be the first normal map and then doesn't flip through the 39 other frames.

**Comment by Tenome** [comment]
I think Tulip reverse also had water that wasn't animated. Maybe all reverse course water is messed up for some reason?

**Comment by Tenome** [comment]
I've confirmed that the reverse water is correctly animated in the PS2 version.

**Comment by k0pp0** [comment]
So in theory the code could be added in the same way as the normal course?

**Comment by Zoukambada** [comment]
@k0pp0 you are right... I was going to create an issue about the water texture bug. This is what I captured last night:
![20241031192008_1](https://github.com/user-attachments/assets/9ece70c0-e154-45b3-bf13-56163964e14a)
